api/onboarding/urls.py

```diff
@@ -0,0 +1,25 @@
+from common.core.utils import is_oss
+from django.urls import path
+
+from onboarding.views import (
+    ReceiveSupportRequestFromSelfHosted,
+    send_onboarding_request_to_saas_flagsmith_view,
+)
+
+app_name = "onboarding"
+
+urlpatterns = [
+    path(
+        "request/receive/",
+        ReceiveSupportRequestFromSelfHosted.as_view(),
+        name="receive-onboarding-request",
+    ),
```

Comment:
Shouldn't this only be added if `if_saas()` is true?

Reply:
Yeah, I didn't do that because it would just return a 400 in all the other environments anyway, but I don't see any reason to include it either.